### PR TITLE
Fix scanning of log output in GetIpAddress

### LINKED ISSUE #80: Regression in system tests
Seeing this on master as of friday; taking note so I can fix it later: https://gist.github.com/erikh/118796e67335899d3b81

### PR DESCRIPTION
**Pull request by erikh**
Resolves #80.

### CODE REVIEW

File: systemtests/utils/utils.go [hunk 1]
```diff
@@ -296,7 +295,20 @@ func GetIpAddress(t *testing.T, node TestbedNode, ep string) string {
...
-	return string(output)
+
+	for _, str := range strings.Split(string(output), "\\n\\t") {
+		if strings.HasPrefix(str, "IpAddress") {
+			ret := strings.SplitN(str, ":", 2)
+			if len(ret) < 2 {
+				break
+			}
+
+			ret = strings.Split(ret[1], ",")
```
Comment (jainvipin) [inline systemtests/utils/utils.go]: Minor: Instead of split strings.TrimPrefix(contName, ",") would be more explicit

Comment (jainvipin) [inline systemtests/utils/utils.go]: sorry it would have to be TrimRight, not TrimPrefix...
otherwise changes LGTM

**Comment by erikh** [comment]
This should pass and has the simplification with `TrimRight` you have suggested.

**Comment by jainvipin** [comment]
LGTM

**Comment by mapuri** [comment]
ah, ok..this explains the sanity failures :relaxed: 

LGTM, can merge once sanity passes

**Pull request merged**